packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1726,6 +1770,11 @@ export class ContainerRuntime
 		this.updateDocumentDirtyState(newState);
 	}
 
+	private async applyStashedIdAllocationOp(content: any) {
+		this.idCompressor = IdCompressor.deserialize(content.stashedState);
```

Comment:
Shouldn't you check or validate that id compression is enabled? This is done in all the other instances where id compressor is used.

Reply:
This is being checked now in `applyStashedOp` before calling `applyStashedIdAllocationOp`